portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/tester")
```

Comment:
I have a doubt to when it comes to the @WebServlet("/tester") line. I can put any path here and it works if I enter it manually in the address bar, but if I use one of my html files it just overwrites it; to where should I point the servlet so it "forms part" of my portfolio directly?

Reply:
How are you putting it into your HTML file? Try sending a PR for all of projet1 step1.

Reply:
It would be helpful to see how you put the servlet in the HTML file. 
For an example, this is how I did mine
`        <h3><a  href="/rand_response">Click Here</a> to get a random message</h3> ` 
inside my index.html file where `"/rand_reponse"` is is in my `@WebServlet("/rand_response")`

Reply:
@GoogleyGabriel, I haven't implemented it on my HTML file because I was not sure how to do so, but I'll get working on that. Thank you @SamRod33, that helps!

Reply:
Yup! +1 to what sam said. It seems like your other PR makes more sense -- i would close this one and just use the other one.